src/metatensor/models/experimental/alchemical_model/tests/test_regression.py

```diff
@@ -21,6 +21,9 @@
 from . import DATASET_PATH
 
 
+torch.set_default_dtype(torch.float64)
+
+
```

Comment:
I'd like not to see this at the top, because pytest executes this code at the beginning of the test session, and so this will be applied to all the tests (not only those in this file). If you need this, try:

```
def test_your_test():
  torch.set_default_dtype(torch.float64)
  # ...
  # your tests and asserts
  torch.set_default_dtype(torch.float32)